### DISCUSSION TITLE
It's not working.

### DISCUSSION THREAD

**Discussion by shchan618** [Rapptz/discord.py#6402] [Q&A]
Run setup.py and it will run shortly and turn off immediately. Python version is 3.6.2.

**Comment by mesub7** [comment]
Could you give us some more information to work off?

Code and error messages would be extremely valuable here.

Also whatever you have is called `setup` which may be why it turns off not too long afterwards. Alternatively, you may be invoking the python file incorrectly.

**Reply by shchan618** [reply]
setup.py exited immediately and did not see any error messages

**Reply by mesub7** [reply]
How are you invoking the file?

**Reply by bandwiches** [reply]
Sounds like you're trying to install the package from scratch, do not recommend.

**Comment by JavascriptLearner815** [comment]
(This technically isn't an answer, as the file isn't included nor a project or code containing the file is included.)

I'd suggest putting this as a bug issue instead of Discussions, especially because of templates. Just the title is sketchy. At least use a template or add the codefile before we can answer.

**Reply by Harmon758** [reply]
This is unlikely to be an issue with discord.py itself, so a discussion is more appropriate than an issue.

@shchan618 simply need to provide more information for anyone to be able to help them, e.g. what's "not working" and what behavior they expect.

The assumption would be that setup.py refers to the one for discord.py.

**Comment by Alawal** [comment]
Hello,

with problems like this more details would be good.
Do you encounter any errors during startup that you can share with us? 
Have you checked the code (tokens, syntax correctness)?

A simple _"It's not working."_ and giving a filename where we neither know the content nor the functions doesn't help.

ARealWant
